math/extended_euclids_algorithm.cpp

```diff
@@ -0,0 +1,33 @@
+#include <bits/stdc++.h>
+
+using namespace std;
+
+int gcdExtended(int a, int b, int& x, int& y)
+{
+    if (a == 0)
+    {
+        x = 0;
+        y = 1;
+        return b;
+    }
+
+    int xTemp, yTemp;
```

Comment:
Same as in main. Assing useful values to this variables. Otherwise it contains random chunk